test/fixtures-setupsh/install-sysctl.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if [ $(command -v sysctl) ]; then
+    exit 0
+fi
+
+# Detect apt
+if [ $(command -v apt-get) ]; then
+    exit 1
+fi
+
+# Detect dnf
+if [ $(command -v dnf) ]; then
+    dnf install -y procps
+    exit 0
+fi
+
+# Detect yum
+if [ $(command -v yum) ]; then
+    exit 1
```

Comment:
do we support yum?

Reply:
this is a test and yes we test it under fedora

Reply:
good question.